angular/pxb-component-theme.scss

```diff
@@ -94,11 +94,11 @@
             background-color: map-get($background, background);
         }
 
-        .pxb-info-list-item-active .pxb-info-list-item .mat-list-item-content {
+        .pxb-drawer-nav-item-active {
             color: map-get($primary, 500);
         }
 
-        .pxb-info-list-item-active:after {
+        .pxb-drawer-nav-item-active-highlight {
             background-color: map-get($primary, 50);
         }
```

Comment:
Should we add a changelog entry about this? Is changing the highlight implementation a breaking change?

Reply:
This is _technically_ a breaking change since the class name used to adjust the highlight color is being changed.  

I think if we add a comment in the release notes we should be alright.

Reply:
Or you know what we could do to prevent a breaking change in the theme is to temporarily support the old class styles and new ones.  When we have our next major release we can remove the deprecated class names.  How does that sound?

Reply:
Sounds like a good application for _Release Planning_. I'm fine with it how it is (I don't think there's anybody out there customizing the styles of the selected drawer item right now), but let's mention the rename in the changelog just in case.